packages/azure/README.md

```diff
@@ -0,0 +1,114 @@
+# Azure ARM Templates
+
```

Comment:
It would be useful to include a pre-built template.json so that users can perform a one-click deploy to Azure from GitHub, similar to this repo: https://github.com/Azure/azure-quickstart-templates/tree/master/101-vm-simple-linux/

Reply:
oh, good idea.